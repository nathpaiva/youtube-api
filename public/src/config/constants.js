export const API_KEY = 'AIzaSyCwChe2uQJtgQQuMdTdzJFgDC3J3Bx35Po';
export const MIN_INCREMENTS = 4;
export const MAX_RESULTS = 4;
export const CHANNEL_ID = 'UC9d_htYmYYFUXB2vBKx4NgA';
export const API_URL = 'https://www.googleapis.com/youtube/v3/search';
export const API_URL_VIEWS = 'https://www.googleapis.com/youtube/v3/videos';
