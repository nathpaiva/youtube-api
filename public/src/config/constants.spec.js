'use stict';

import React from 'react';
import renderer from 'react-test-renderer';
import * as YT from './constants';


describe('#Test value of constants', () => {
  it('#Should check API_KEY', () => {
    const response = YT.API_KEY;

    expect(response === 'AIzaSyCwChe2uQJtgQQuMdTdzJFgDC3J3Bx35Po').toEqual(true);
    expect(response).toMatchSnapshot();
  });

  it('#Should check MIN_INCREMENTS', () => {
    const response = YT.MIN_INCREMENTS;
    expect(response === 4).toEqual(true);

    expect(response).toMatchSnapshot();
  });

  it('#Should check MAX_RESULTS', () => {
    const response = YT.MAX_RESULTS;
    expect(response === 4).toEqual(true);

    expect(response).toMatchSnapshot();
  });

  it('#Should check CHANNEL_ID', () => {
    const response = YT.CHANNEL_ID;
    expect(response === 'UC9d_htYmYYFUXB2vBKx4NgA').toEqual(true);

    expect(response).toMatchSnapshot();
  });

  it('#Should check API_URL', () => {
    const response = YT.API_URL;
    expect(response === 'https://www.googleapis.com/youtube/v3/search').toEqual(true);

    expect(response).toMatchSnapshot();
  });

  it('#Should check API_URL_VIEWS', () => {
    const response = YT.API_URL_VIEWS;
    expect(response === 'https://www.googleapis.com/youtube/v3/videos').toEqual(true);

    expect(response).toMatchSnapshot();
  });
});
