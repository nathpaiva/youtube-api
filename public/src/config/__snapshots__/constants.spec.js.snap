// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`#Test value of constants #Should check API_KEY 1`] = `"AIzaSyCwChe2uQJtgQQuMdTdzJFgDC3J3Bx35Po"`;

exports[`#Test value of constants #Should check API_URL 1`] = `"https://www.googleapis.com/youtube/v3/search"`;

exports[`#Test value of constants #Should check API_URL_VIEWS 1`] = `"https://www.googleapis.com/youtube/v3/videos"`;

exports[`#Test value of constants #Should check CHANNEL_ID 1`] = `"UC9d_htYmYYFUXB2vBKx4NgA"`;

exports[`#Test value of constants #Should check MAX_RESULTS 1`] = `4`;

exports[`#Test value of constants #Should check MIN_INCREMENTS 1`] = `4`;
